packages/blockchain-wallet-v4-frontend/src/data/components/interest/sagas.ts

```diff
@@ -66,16 +66,14 @@ export default ({ api }: { api: APIType }) => {
   }: ReturnType<typeof A.initializeInterest>) {
     let defaultAccountR
 
-    switch (payload.coin) {
-      case 'BTC':
-        const accountsR = yield select(
-          selectors.core.common.btc.getAccountsBalances
-        )
-        const defaultIndex = yield select(
-          selectors.core.wallet.getDefaultAccountIndex
-        )
-        defaultAccountR = accountsR.map(nth(defaultIndex))
-        break
+    if (payload.coin === 'BTC') {
```

Comment:
Why did you change this from a switch statement?

Reply:
Was hitting a TS error. I think it was because I didn't have a default. Should I change it back to a switch statement and just break on default?

Reply:
Yes please